src/ops/ops.go

```diff
@@ -566,3 +567,14 @@ func (o *ops) CreateRandomHostname(hostname string) error {
 func (o *ops) GetHostname() (string, error) {
 	return os.Hostname()
 }
+
+func (o *ops) CreateManifests(kubeconfig string, manifestFilePath string) error {
+	command := fmt.Sprintf("oc --kubeconfig=%s apply -f %s", kubeconfig, manifestFilePath)
+	output, err := o.ExecCommand(o.logWriter, "bash", "-c", command)
+	if err != nil {
+		return err
```

Comment:
will we have log in case of failure? i mean if ExecCommand is logging failure?

Reply:
No. Thanks, will add.